modules/Assay/Platform.csv

```diff
@@ -5,44 +5,44 @@ Orbitrap Fusion Lumos Tribrid,Thermo Scientific Orbitrap Fusion Lumos Tribrid Ma
 Spectramax M Series,A Spectramax M Series Microplate Reader,,,FALSE,https://www.moleculardevices.com/products/microplate-readers/multi-mode-readers/spectramax-m-series-readers,platform,,,,Spectramax_M_Series,Class,Assay,Platform,Platform,,,
 LI-COR Odyssey CLx,A LI-COR Odyssey CLx imaging system,,,FALSE,https://www.licor.com/bio/odyssey-clx/,platform,,,,LI-COR_Odyssey_Clx,Class,Assay,Platform,Platform,,,
 Q Exative HF,Thermo scientific Q Exative,,,FALSE,http://purl.obolibrary.org/obo/MS_10025223,platform,,,,Q_Exative_HF,Class,Assay,Platform,Platform,,,
-HiSeq3000,Illumina HiSeq 3000,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21290,platform,,,,HiSeq3000,Class,Assay,Platform,Platform,,,
-HiSeq2500,Illumina HiSeq 2500,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL16791,platform,,,,HiSeq2500,Class,Assay,Platform,Platform,,,
-HiSeq4000,Illumina HiSeq 4000,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL20301,platform,,,,HiSeq4000,Class,Assay,Platform,Platform,,,
-NextSeq500,Illumina NextSeq 500,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL18573,platform,,,,NextSeq500,Class,Assay,Platform,Platform,,,
-NextSeq 550,Illumina NextSeq 550,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21697,platform,,,,NextSeq 550,Class,Assay,Platform,Platform,,,
-NextSeq 1000,Illumina NextSeq 1000,,,FALSE,,platform,,,,NextSeq 1000,Class,Assay,Platform,Platform,,,
-NextSeq 2000,Illumina NextSeq 2000,,,FALSE,,platform,,,,NextSeq 2000,Class,Assay,Platform,Platform,,,
-HiSeq2000,Illumina HiSeq 2000,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL11154,platform,,,,HiSeq2000,Class,Assay,Platform,Platform,,,
-MiSeq,Illumina MiSeq,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL15520,platform,,,,MiSeq,Class,Assay,Platform,Platform,,,
+Illumina HiSeq 3000,Illumina HiSeq 3000,,,FALSE,https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21290,platform,,,,HiSeq3000,Class,Assay,Platform,Platform,,,
```

Comment:
This is great! Do we need to remap values in the portal as well?

Reply:
Yep, can track that in the internal JIRA issue.